src/frontend/apps/desk/src/features/mail-domains/components/ModalCreateMailbox.tsx

```diff
@@ -88,27 +93,47 @@ export const CreateMailboxForm = ({
     },
   });
 
+  const errorCauses = useAPIError({
+    error,
+    errorParams: {
+      local_part: {
+        causes: ['Mailbox with this Local_part and Domain already exists.'],
+        handleError: () => {
+          methods.setError('local_part', {
+            type: 'manual',
+            message: t('This email prefix is already used.'),
+          });
+          methods.setFocus('local_part');
+        },
+      },
+      secret: {
+        causes: [
+          "Please configure your domain's secret before creating any mailbox.",
+          `Please check secret of the mail domain ${mailDomain.slug}`,
+        ],
```

Comment:
I did not write tests for this part as Marie is working on a modification to replace these two errors.